gnomad/resources/grch37/reference_data.py

```diff
@@ -96,6 +96,23 @@
     },
 )
 
+clinvar_pathogenic = VersionedTableResource(
+    default_version="20181028",
+    versions={
+        "20181028": TableResource(
+            path="gs://gnomad-public/resources/grch37/clinvar/clinvar_pathogenic_20181028.ht",
+            import_func=import_sites_vcf,
+            import_args={
+                "path": "gs://gnomad-public/resources/grch37/clinvar/clinvar_pathogenic_20181028.vcf.bgz",
```

Comment:
Goes with comment above. This would keep us from needing to export the vcf and then reimport it. Actually, I just got thinking that maybe the best way to do this is have a single clinvar resource, but add a pathogenic annotation to that resource on loading. That way we only store one table, but have the pathogenic info already annotated